### ISSUE TITLE
Add capability to run non tagged version of k8s

### ISSUE THREAD

**Issue by chrislovecnm** [TheNewNormal/kube-solo-osx#60]
Any way to run master on this awesome piece of software??

**Comment by rimusz** [comment]
@chrislovecnm it does have master services and node ones as well

**Comment by chrislovecnm** [comment]
Let me see if I can ask this question better... Can I run the latest source code off of the master branch on k8s github repo?  Bleeding edge 😄 

```
git clone https://github.com/kubernetes/kubernetes
hack/build-go.sh
```

Run those binaries

**Comment by rimusz** [comment]
I see, any prebuilt version from https://github.com/kubernetes/kubernetes/releases can be installed via 'Updates->Change Kubernetes version'
So basically you want to use locally built binaries, hmm it should be possible to automate, just need to find a time for that.
As a work around for now you can built binaries, copy OS X `kubectl` to `/kube-solo/kube` and update `/kube-solo/kube/kube.tgz` with your freshly built linux binaries, then via `Setup->Destroy Kube-Solo VM` and on next  `Up` the new VM will be created with kubernetes version in kube.tgz file

**Comment by chrislovecnm** [comment]
@rimusz WOOP!!

**Comment by chrislovecnm** [comment]
FREE COOKIE FOR YOU!!

**Comment by rimusz** [comment]
that `hack/build-go.sh` takes ages on my mac mini to run

**Comment by chrislovecnm** [comment]
`<facepalm>`

**Comment by rimusz** [comment]
@chrislovecnm take a look to https://github.com/TheNewNormal/kube-solo-osx/blob/master/src/functions.sh#L270-L338 and if you have time to hack a new function to build form source the git gist will be very welcome, I will do the rest to include it to the App

**Comment by rimusz** [comment]
fixed by #61
